player/command.c

```diff
@@ -3797,6 +3798,7 @@ static const struct m_property mp_properties[] = {
     {"video-format", mp_property_video_format},
     {"video-frame-info", mp_property_video_frame_info},
     {"video-codec", mp_property_video_codec},
+    {"video-performance", mp_property_video_performance},
```

Comment:
(If we're naming it vo-performance then I'll also sort it to the position of the other vo-* options, e.g. vo-configured)